webpack/assets/javascripts/react_app/components/users/personalAccessTokens/Token.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import TimeAgo from '../../common/TimeAgo';
+
+/* eslint-disable camelcase */
+export default ({ id, name, created_at, expires_at, last_used_at, user_id, revocable }) =>
+  <tr>
+    <td>
+      {name}
+    </td>
+    <td>
+      <TimeAgo date={created_at} />
+    </td>
+    <td>
+      <TimeAgo date={expires_at} />
+    </td>
+    <td>
+      <TimeAgo date={last_used_at} />
+    </td>
+    <td>
+  {revocable &&
+      <span className="btn btn-sm btn-default">
+        <a
+          data-confirm="Are you sure?"
+          data-id="aid_personal_access_tokens_1_revoke"
```

Comment:
this will use jquery delete dialog ("Are you sure") - we should probably have an alternative option?